drill/README.md

```diff
@@ -53,6 +53,7 @@ apache drill 1.9.0
 +---------------------+
 12 rows selected (3.943 seconds)
 ```
+In order to test Drill on single node cluster it is needed to use drill-embedded version.
```

Comment:
(Since I'm not really familiar with Drill): shouldn't the init action start drill in embedded mode?

Reply:
Yes it seems to be running in embedded mode. 
Drill-embedded script just fires sqlline with local zookeeper specified.